### ISSUE TITLE
Can't build with chakra-ui/utils latest

### ISSUE THREAD

**Issue by KannarFr** [anubra266/choc-autocomplete#267]
Hi,

I had to downgrade @chakra-ui/utils to v2.0.8 to make a project using this library to build.

```
error during build:                                                                                                                                                                                                                                                                                                                                         
node_modules/@choc-ui/chakra-autocomplete/dist/index.es.js (5:86): "getNextItem" is not exported by "node_modules/@chakra-ui/utils/dist/esm/index.mjs", imported by "node_modules/@choc-ui/chakra-autocomplete/dist/index.es.js".
file: /home/kannar/git/myle/web/node_modules/@choc-ui/chakra-autocomplete/dist/index.es.js:5:86
                                                                                       
3: import { createContext as lo } from "@chakra-ui/react-utils";
4: import { useDisclosure as co, useControllableState as uo, useUpdateEffect as fo, forwardRef as Sr, Popover as po, cha...
5: import { pick as wo, isDefined as cn, isEmpty as Rt, runIfFn as Z, isUndefined as Vr, getNextItem as Ro, getPrevItem ...
                                                                                         ^
6: import Vt from "@emotion/styled";                          
7: import { ThemeContext as Po } from "@emotion/react";       
                                                                                       
    at getRollupError (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/parseAst.js:395:41)
    at error (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/parseAst.js:391:42)
    at Module.error (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:15535:16)
    at Module.traceVariable (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:15984:29)
    at ModuleScope.findVariable (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:13770:39)
    at FunctionScope.findVariable (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:5252:38)
    at FunctionBodyScope.findVariable (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:5252:38)
    at Identifier.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:5035:40)
    at CallExpression.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:2855:23)
    at CallExpression.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:11235:15)
    at VariableDeclarator.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:2855:23)
    at VariableDeclaration.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:2851:28)
    at BlockStatement.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:2851:28)
    at FunctionDeclaration.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:2855:23)
    at Program.bind (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:2851:28)
    at Module.bindReferences (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:15514:18)
    at Graph.sortModules (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:21019:20)             
    at Graph.build (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:20922:14)
    at async file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:21607:13
    at async catchUnfinishedHookActions (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:21076:16)
    at async rollupInternal (file:///home/kannar/git/myle/web/node_modules/rollup/dist/es/shared/node-entry.js:21602:5)
    at async build (file:///home/kannar/git/myle/web/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:65386:14)
    at async CAC.<anonymous> (file:///home/kannar/git/myle/web/node_modules/vite/dist/node/cli.js:828:5)
```

**Comment by Kysluss** [comment]
Hi @KannarFr 

On vacation this week, but should be able to take a look when I get back.  Is this a fresh install of react with chakra and autocomplete or did you have to install anything specific?

Reason I ask, I just released a new version the other week that had all the dependencies upgraded and I didn't get this error when I ran through some tests.

**Comment by KannarFr** [comment]
Hi,

I tried to bump to bump because I had other issues with dependencies. After all, we should use @chakra-ui/react and not other sub-packages like @chakra-ui/system, etc.

But it didn't fixes and downgrade @chakra-ui/utils fixed the issue.

**Comment by Kysluss** [comment]
Thank you.  This should be resolved in [v5.7.0](https://github.com/anubra266/choc-autocomplete/releases/tag/v5.7.0).  Thanks to @anubra266 for fixing that.
